src/main/java/net/openhft/chronicle/core/util/StringUtils.java

```diff
@@ -99,11 +99,20 @@ public static boolean isEqual(@Nullable StringBuilder s, @Nullable CharSequence
         if (cs == null) return false;
         int length = cs.length();
         if (s.length() != length) return false;
-        char[] chars = StringUtils.extractChars(s);
-        for (int i = 0; i < length; i++)
-            if (chars[i] != cs.charAt(i))
-                return false;
-        return true;
+
```

Comment:
Please add a unit test to StringUtilsTest to exercise these code paths.

Reply:
@epickrram I have added the test.